static/src/stylesheets/module/_discussion.scss

```diff
@@ -840,13 +840,27 @@ $comment-recommend-button-size: 19px;
 .d-comment__actions {
     position: absolute;
     bottom: $gs-baseline;
+    opacity: .7;
     .discussion--closed & {
-        display: none;
+        .d-comment__action--reply, .d-comment__action-separator, .d-comment__action-pick {
+            display: none;
+        }
+        .sharing-component {
+            margin: 0 !important;
```

Comment:
I agree with @gtrufitt, if you find yourself using `!important` it is likely some code should be refactored. Ideally we shouldn't be "overriding" any styles.